src/options.ts

```diff
@@ -15,3 +23,26 @@ document.addEventListener('DOMContentLoaded', e => {
       chrome.storage.local.get('activeFeatures', result => { ... })
     */
 });
+
+canvasTextbox.onfocus = () => {
+    canvasTextbox.setAttribute('type', 'text');
+};
+
+canvasTextbox.onblur = () => {
+    canvasTextbox.setAttribute('type', 'password');
+    chrome.storage.local.set({tokens: {canvas: canvasTextbox.value}});
+};
+
+//disable canvas API token text box when Canvas Integration
+//is not desired
+canvasCheckbox.onchange = () => {
+    if (canvasCheckbox.checked) {
+        canvasTextbox.readOnly = false;
+        canvasTextbox.style.backgroundColor = '#FFFFFF';
+        canvasTextbox.style.color = '#000000';
+        return;
```

Comment:
Any reason for implicit return here rather than an else statement?

Reply:
Prettier doesn't seem to like it much when I write it with an explicit else statement (it reformats it this way). There are actually a couple rules that Prettier follows that the linter doesn't like either, I was wondering if we could adjust one or the other so that they are consistent, just so we have less squiggles.

Reply:
Ah, this is an ESLint rule called [`no-else-return`](https://eslint.org/docs/rules/no-else-return). If you're following those examples and it's still giving you problems, you can make a change to `.eslintrc.json` to remove the `no-else-return` rule. I might have misunderstood its usage.